net-misc/dropbear/dropbear-2018.76-r1.ebuild

```diff
@@ -0,0 +1,108 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+inherit pam savedconfig user
+
+DESCRIPTION="small SSH 2 client/server designed for small memory environments"
+HOMEPAGE="https://matt.ucc.asn.au/dropbear/dropbear.html"
+SRC_URI="https://matt.ucc.asn.au/dropbear/releases/${P}.tar.bz2
+	https://matt.ucc.asn.au/dropbear/testing/${P}.tar.bz2"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~m68k ~mips ~ppc ~ppc64 ~s390 ~sh ~sparc ~x86 ~amd64-linux ~x86-linux"
+IUSE="bsdpty minimal multicall pam +shadow static +syslog +utmp zlib"
+
+LIB_DEPEND="zlib? ( sys-libs/zlib[static-libs(+)] )
+	dev-libs/libtommath[static-libs(+)]"
+RDEPEND="!static? ( ${LIB_DEPEND//\[static-libs(+)]} )
+	pam? ( virtual/pam )"
+DEPEND="${RDEPEND}
+	static? ( ${LIB_DEPEND} )"
+RDEPEND+=" pam? ( >=sys-auth/pambase-20080219.1 )"
+
+REQUIRED_USE="pam? ( !static )"
+
+set_options() {
+	secondary_progs=(
+		dbclient dropbearkey
+		$(usex minimal "" "dropbearconvert scp")
+	)
+	progs=(dropbear "${secondary_progs[@]}")
+	makeopts=(
+		MULTI=$(usex multicall 1 0)
+		STATIC=$(usex static 1 0)
+	)
+}
+
+src_prepare() {
+	eapply "${FILESDIR}/${PN}-2018-dbscp.patch"
```

Comment:
I hinted this earlier, but let's clarify: You don't need to add a new identical patch. Just call `eapply -p0 "${FILESDIR}/${PN}-0.46-dbscp.patch"` here. 

Also, if you make a patch that's appliable by default eapply, you can initiate a `PATCHES` array and it will get patched here. Also, `default` should be the first thing to call in `src_prepare` (sorry just noticed it now).

Reply:
I would prefere the `PATCHES` way...